### ISSUE TITLE
Create factory programs and Presets

### ISSUE THREAD

**Issue by GoogleCodeExporter** [shaffer420/midi-shape-shifter#22]
```
Create a set of factory programs and presets that demonstrate the functionality 
of the program.
```

Original issue reported on code.google.com by `RobIan...@gmail.com` on 4 Nov 2012 at 11:03
- Blocking: #21

**Comment by GoogleCodeExporter** [comment]
Original comment by `RobIan...@gmail.com` on 4 Nov 2012 at 11:04
- Now blocking: #21

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 02eb36baf8bd.
```

Original comment by `RobIan...@gmail.com` on 19 Feb 2013 at 2:01
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
Original comment by `RobIan...@gmail.com` on 10 Mar 2013 at 10:59
- Changed state: **Verified**
